pytorch/training/docker/1.10/py3/cu113/Dockerfile.sagemaker.gpu

```diff
@@ -9,9 +9,9 @@ LABEL dlc_major_version="1"
 ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.10
 ARG PYTHON_SHORT_VERSION=3.8
-ARG CONDA_PREFIX=/opt/conda
 ARG METIS=metis-5.1.0
 ARG RMM_VERSION=0.15.0
+ARG MAMBA_VERSION=4.12.0-0
```

Comment:
nit: is this arg used in this file?

Reply:
looks like it wasn't, good catch, I will remove